adaptive_alerting_detector_build/detectors/constant_threshold.py

```diff
@@ -173,6 +180,16 @@ def _train_quartile(self, sample):
         )
 
 
+def _threshold_type(metric_type):
+    mappings = {
+        "REQUEST_COUNT": ConstantThresholdType.TWO,
```

Comment:
This one is a little iffy to me -- LEFT seems like a better default. But I can see that we might want two tails for bot spikes or a huge request volume stemming from some accidentally generous offer. Your call.

Reply:
I'll leave it as is for now. We can adjust if we discover this was not the right choice.

Reply:
For reference, here's where the tail type mapping were decided: https://jira.expedia.biz/browse/AUT-2457